src/en/components/top-nav/design.md

```diff
@@ -6,4 +6,38 @@ tags: ['topnavEN', 'design']
 date: "git Last Modified"
 ---
 
-## Design
+## Top navigation anatomy – on desktop
+
+<ol class="anatomy-list">
+  <li>The <strong>home nav link</strong> identifies the site using text or a logo and it links to the home page. It uses a nav link that is added within the <code>home</code> slot.</li>
+  <li>The <strong>nav link</strong> is a link to a popular page or section on the site. Nav links can be in the top level if they're standalone.</li>
+  <li>The <strong>nav group</strong> can be collapsed and expanded and generally contains a list of 2-5 links.</li>
+  <li>The <strong>nav group label</strong> describes the category of links in that group. It acts as a button to expand and collapse the group of links.</li>
+  <li>The <strong>nav group chevron icon</strong> points up when the nav group is collapsed and down when it's expanded.</li>
+  <li>The <strong>nav link with active indicator</strong> gives visual prominence to the current page.</li>
+</ol>
+
+<img class="b-sm b-default p-400" src="/images/en/components/anatomy/gcds-top-nav-anatomy.svg" alt="Top navigation anatomy showing page navigation with a dark blue box for the logo or home link followed by two greyed boxes, the last one having a dark blue line underneath to show the current page." />
+
+## Design and accessibility for top navigation
+
+### Set the site name
```

Comment:
Will we need to do a slight update to this section since we no longer have `site name` included in the anatomy image?

Reply:
Good call, I'll update the heading to `Set the home nav link`